docs/chain/collator.md

```diff
@@ -530,8 +530,18 @@ For **peregrine**, the parachain bootnodes are:
 
 ## Lifecycle of a Collator
 
-<Mermaid
-chart={`flowchart TD
+When you intent to stop collating or just being a collator candidate, you have to go through three stages until your staked tokens are unlocked and your collator state is purged from the chain.
+
+First, you signal your intent by calling `init_leave_candidates`. You will both be removed from the `CandidatePool` and your state switches from `Active` to `Leaving(leave_round)`, where `leave_round` reflects the number of sessions in which you can actually leave. A leaving candidate cannot be selected as an active collator for the sessions hereinafter. Moreover, you cannot receive new Delegations and existing Delegations cannot be adjusted. However, Delegations can still be revoked.
```

Comment:
Maybe add that rewards are still payed out for the current and next session for the collator.

Reply:
[72d0101](https://github.com/KILTprotocol/docs/pull/64/commits/72d010156e160e3e21c394e1b7b9eaf2611b883a)